evalml/tests/component_tests/test_estimators.py

```diff
@@ -381,3 +381,50 @@ def test_estimator_feature_importance(
 
     assert not estimator.feature_importance.isna().any()
     assert len(X.columns) == len(estimator.feature_importance)
+
+
+@pytest.mark.parametrize("estimator_class", _all_estimators())
```

Comment:
might be cleaner to use `get_estimators(ProblemTypes.TIME_SERIES_REGRESSION):` instead of `_all_estimators` here so that we don't have empty test cases.

Reply:
Good call!